lib/index.d.ts

```diff
@@ -0,0 +1,600 @@
+import { OrderItem } from "./parsers/orderItems";
+
+type InitParams = {
+    accessKeyId: string;
+    authToken?: string;
+    host?: string;
+    merchantId: string;
+    port?: number;
+    region?: string;
+    secretAccessKey: string;
+};
+
+// cSpell: disable
+type MWS_MARKETPLACES = {
```

Comment:
This is a nice ENUM to export. Whenever a user needs a marketplace ID, the user could simply call this enum, e.g.  `MWS_MARKETPLACES.CA`.

Reply:
````
import MWS from '@ericblade/mws-advanced';

MWS.MWS_MARKETPLACES.CA;
````
should work, it's exported for Javascript consumers as well.

is that good?  the "export = " syntax that seems to be required to get it to all function with IDEs with both Javascript and Typescript consumers doesn't allow exporting of anything else .. i'm not sure exactly what that syntax *does* do, but i know i can't add any other exports to the file.

Reply:
@pwrnrd what do you think about the existing solution, or do you know of a way to improve upon it, given what we have here?